web/templates/settings/invites_list_admin.hbs

```diff
@@ -1,4 +1,4 @@
-<div id="admin-invites-list" class="settings-section" data-name="invites-list-admin">
+<div id="admin-invites-list" class="settings-section" data-user-settings-section="invites-list-admin">
```

Comment:
If I remove that extra div i.e ```<div class="user-section-container" data-user-settings-section="user-list-admin">``` , then I need to have some class (```#users-admin-panel .settings-section``` in this case) which can be used to hide the user section on toggle.
Presently I use ```user-section-container``` to hide.

Reply:
Yeah, we need some common class but we can name it like `user-settings-section` instead of `settings-section` because that is used for each setting section opened from left panel.

Reply:
Sure , updated .